content/spin/v2/key-value-store-tutorial.md

```diff
@@ -157,64 +157,62 @@ Now let's use the Spin SDK to:
 {{ startTab "Rust"}}
 
 ```rust
-use http::{Method, StatusCode};
 use spin_sdk::{
-    http::{IntoResponse, Response},
+    http::{IntoResponse, Request, Response, Method},
     http_component,
     key_value::Store,
 };
 
 #[http_component]
-fn handle_request(req: http::Request<Vec<u8>>) -> anyhow::Result<impl IntoResponse> {
+fn handle_request(req: Request) -> anyhow::Result<impl IntoResponse> {
     // Open the default key-value store
     let store = Store::open_default()?;
 
     let (status, body) = match *req.method() {
-        Method::POST => {
+        Method::Post => {
             // Add the request (URI, body) tuple to the store
-            store.set(req.uri().path(), req.body().as_slice())?;
+            store.set(req.path(), req.body())?;
             println!(
                 "Storing value in the KV store with {:?} as the key",
-                req.uri().path()
+                req.path()
             );
-            (StatusCode::OK, None)
```

Comment:
just curious why are we changing this?

Reply:
@rajatjindal It doesn't work out of the box with the current template because that no longer references `http` by default.